### DISCUSSION TITLE
Monochrome tray icon option

### DISCUSSION THREAD

**Discussion by Sadi58** [cstayyab/WALC#214] [Feature Requests]
It would be great if users could choose between regular (color) tray icon(s) and monochrome.
The present one looks a bit out of place when monochrome icons are used in the panel as seen in the screenshot below.
![combined-image_1669464060](https://user-images.githubusercontent.com/6375934/204087964-b1d9da21-0fb6-482b-a246-487bc4a913c0.png)

**Comment by albakhtari** [comment]
Dude, do you know how long it took me to get that logo approved by @Zzombiee2361?!
I don't care if doesn't suit your panel!

<h1>:rofl:</h1>

Just kidding, I was kind of thinking about it myself... It doesn't look too good in colour on the system tray...

![image](https://user-images.githubusercontent.com/79008923/204089118-fc9e9ed6-9741-457a-9986-9e2c4f9271de.png)

When I have time, I'll create the monochrome version of the logo and open a PR, then it'll be @Zzombiee2361's decision on what to do next.

I think an option to change the icon type should be added to the walc dashboard...

**Reply by Sadi58** [reply]
I know! I've seen you've put so much effort into this project, and it really deserves it! And thanks to @Zzombiee2361 being kind enough to react positively to all such contribution efforts, it's getting better and better! 😉

**Reply by albakhtari** [reply]
Created the PR (#215)

**Reply by albakhtari** [reply]
![image](https://user-images.githubusercontent.com/79008923/204124200-71ef1653-be78-4771-a6c7-c4e64f609982.png)

I was thinking, maybe something like this :point_up: ?

I'll add the light logos to the PR too.

**Reply by Sadi58** [reply]
I think coding related to this should better take into account that some panels/docks can use one monochrome icon and change it between dark and light any time needed - e.g. when using a dynamic theme that switches to dark at sunset etc.
So while both versions of monochrome icons will be needed for some, it should also be possible to use one by such smart panels.
I think all this may affect naming of the tray monochrome icons as well as coding, although I'm no expert in these things.
